litmus-portal/graphql-server/pkg/file_handlers/file_handlers.go

```diff
@@ -17,6 +17,9 @@ func FileHandler(w http.ResponseWriter, r *http.Request) {
 	serviceAddr := os.Getenv("SERVICE_ADDRESS")
 	subscriberImage := os.Getenv("SUBSCRIBER_IMAGE")
 	subscriberNS := os.Getenv("DEPLOYER_NAMESPACE")
+	subscriberSC := os.Getenv("SUBSCRIBER_SCOPE")
+	workflowSC := os.Getenv("WORKFLOW_SCOPE")
```

Comment:
Subscriber and workflow will have the same scope and ns. If there is any change for the workflow it will be mentioned in the workflow yaml itself.